debian/control

```diff
@@ -0,0 +1,87 @@
+Source: fuse3
+Section: utils
+Priority: optional
+Maintainer: Laszlo Boszormenyi (GCS) <gcs@debian.org>
+Build-Depends:
+ debhelper (>= 11),
+ pkg-config,
+ meson,
+ udev,
+ python3,
+ python3-pytest
+Standards-Version: 4.3.0
+Homepage: https://github.com/libfuse/libfuse/wiki
+#Vcs-Browser: https://daniel-baumann.ch/gitweb/?p=debian/packages/fuse.git
+#Vcs-Git: git://daniel-baumann.ch/git/debian/packages/fuse.git
+
+Package: fuse3
+Section: utils
+Architecture: linux-any
+Depends:
+ ${misc:Depends},
+ ${shlibs:Depends},
+ libfuse3-3 (= ${binary:Version}),
+ adduser,
+ mount (>= 2.19.1),
+ sed (>= 4),
+ lsb-base (>= 3.2-14)
+Provides: fuse
+Breaks: fuse
+Replaces: fuse
+Description: Filesystem in Userspace (3.x version)
+ Filesystem in Userspace (FUSE) is a simple interface for userspace programs to
+ export a virtual filesystem to the Linux kernel. It also aims to provide a
+ secure method for non privileged users to create and mount their own filesystem
+ implementations.
+
+Package: libfuse3-3
```

Comment:
Why the `-3`?

Reply:
That is what Ubuntu is using.

Reply:
But do we know why?  Just curious.  An opportunity to learn about something that I am not familiar with.  If you don't know off-hand, that's fine.  I can see if I can go dig it up somewhere.

Reply:
I do not know.